src/Symfony/Component/Messenger/Bridge/Amqp/Tests/Transport/AmqpExtIntegrationTest.php

```diff
@@ -32,7 +32,6 @@
 use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
 
 /**
- * @requires extension amqp
```

Comment:
we still need all these annotations, see tests on appveyor (aka when testing symfony/symfony)